src/components/Frames.js

```diff
@@ -25,7 +25,12 @@ function renderFrameTitle({ displayName }: Frame) {
 }
 
 function renderFrameLocation({ source, location }: Frame) {
-  const filename = getFilename(source);
+  const thisSource : ?Source = source;
```

Comment:
Hi @jasonLaster 

Without this check, the flow was producing this error:
<img width="813" alt="screen shot 2017-01-20 at 10 38 51 am" src="https://cloud.githubusercontent.com/assets/4562118/22134198/4953eeca-df00-11e6-865a-56d92e68ab7d.png">

This `if` statement checks if source is undefined, which is possible because Frame's source is declared as `?`. I got this fix from https://flowtype.org/docs/nullable-types.html#_